distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -118,7 +118,7 @@ type File
                         temp.delete_if_exists
 
             ## Attach a warning to the file that it is a dry run
-            warning = Dry_Run_Operation.Warning "Only a dry run has occurred, with data written to a temporary file."
+            warning = Dry_Run_Operation.Warning "Only a dry run has occurred, with data written to a temporary file.  Press the Record Once button ❙⬤❙ to write the actual file."
```

Comment:
Also how do I know as a user what button is 'the Record Once' button?

Reply:
> Also how do I know as a user what button is 'the Record Once' button?

I was hoping the icon would get you there... A tooltip on the button would be useful to confirm the fact though.

Reply:
> > Also how do I know as a user what button is 'the Record Once' button?
> 
> I was hoping the icon would get you there... A tooltip on the button would be useful to confirm the fact though.

Fair, I think the icon is a good pointer. But indeed if we do refer to the button by it's name it would be good to be able to learn somewhere that this is the name (I guess we will be able to see it in tutorials)

Reply:
> Yeah I thought about that as I added it. Similarly for `dry run` which I don't like the name of, but can't currently think of anything better for.
> 
> Do we have a location/mechanism for such constants?

Hm, I'm afraid we don't have a prepared place for this. Sounds like `Visualization` module would be best, but we need to be able to access this from `Base` and `Base` should definitely _not_ depend on `Visualization`.

I'm wondering if we could add the text at the visualization level, but the text depends on context... So I guess that would, at least for now, be over-engineering a simple thing.

I guess I'd just put it somewhere in `Base`.

Reply:
I'm going to merge as is for now as I just wanted a quick fix and the dry run text was already duplicated before this change.

But we should come back and design somewhere for the message constants to go in the future.